eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -32,10 +32,6 @@ steps:
   #       Write-Host 'isWindows: ${{ parameters.isWindows }}'
 
   #       Write-Host 'skipCodeCoverage: ${{ parameters.skipCodeCoverage }}'
-  #       Write-Host 'skipCodeCoverage == true: ${{ eq(parameters.skipCodeCoverage, true) }}'
-  #       Write-Host "skipCodeCoverage == 'true': ${{ eq(parameters.skipCodeCoverage, 'true') }}"
-  #       Write-Host 'skipCodeCoverage != true: ${{ ne(parameters.skipCodeCoverage, true) }}'
-  #       Write-Host "skipCodeCoverage != 'true': ${{ ne(parameters.skipCodeCoverage, 'true') }}"
```

Comment:
What's the purpose of this if commented out? Can we either delete or add a comment on top saying why we want to keep it and how to use it?

Reply:
AzDO has very arcane and hidden ways to interpret some constructs. This code was used for debugging purposes, and it's no longer needed, hence being removed.
The rest of it may still be useful for debugging purposes, it's much easier to uncomment then to try to remember how to write it again.